server_extensions_extra/src/main/resources/batm-extensions.xml

```diff
@@ -1967,5 +1968,17 @@
         <cryptologo cryptocurrency="ILC" file="ilc.png"/>
         <cryptologo cryptocurrency="ILC" file="ilc.svg"/>
     </extension>
+    <extension class="com.generalbytes.batm.server.extensions.extra.cardano.CardanoExtension" >
+        <wallet prefix="cardano" name="Cardano">
```

Comment:
change wallet prefix to name of the wallet service. There will be multiple cardano wallets in future. Naming one "cardano" is unwise. 
change name to Wallet service product name. There will be multiple cardano wallets in future. Naming one "cardano" is unwise.